src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form search-form_big"
+      data-qa="big"
+    >
+      <label class="search-form__label search-form__label_big">
+        <input
+          data-qa="keypress"
+          type="text"
+          placeholder="Try “Los Angeles”"
+          class="search-form__input search-form__input_big"
+        />
+      </label>
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form search-form_small"
```

Comment:
Here again, the indentation is not correct. Please use 2-space offset for the nested elements.